### PR TITLE
Fix the param_count to sum across pipeline_parallel_comm

### PR DESCRIPTION
**Pull request by justusc**

### CODE REVIEW

File: training/dist_clm_train.py [hunk 1]
```diff
@@ -150,7 +153,7 @@ def train_loop(args, pipe, device, train_data_loader, test_data_loader):
 
             if event_reporter is not None and pipe.global_step % epoch_steps == 0:
                 event_reporter.report(object=EventReporter.OBJECT_FINE_TUNE,
-                                      message=f"Epoch competed for step {pipe.global_step}",
+                                      message=f"Epoch competed, at step {pipe.global_step}",
```
Comment (orangetin) [inline training/dist_clm_train.py:156]: completed is misspelled

File: training/dist_clm_train.py [hunk 2]
```diff
@@ -163,11 +166,6 @@ def train_loop(args, pipe, device, train_data_loader, test_data_loader):
                 if dp_rank == 0:
                     save_checkpoint(pipe, args)
 
-                    if event_reporter is not None:
-                        event_reporter.report(object=EventReporter.OBJECT_FINE_TUNE,
-                                              message=f"checkpoint saved for step {pipe.global_step}",
-                                              event_type=EventReporter.EVENT_TYPE_CHECKPOINT_SAVE,
-                                              requires_is_enabled=False)
```
Comment (orangetin) [inline training/dist_clm_train.py:170]: Why are we removing this?

Comment (justusc) [reply training/dist_clm_train.py:170]: This is going to be handled by ft job script.

**Review by orangetin — APPROVED** [review]
LGTM

**Pull request merged**